en/command_line_options.md

```diff
@@ -0,0 +1,36 @@
+# Command Line Options
+
+You can start *QGroundControl* with command line options. These are used to enable logging, run unit tests, and simulate different host environments for testing. 
+
+## Starting QGroundControl with Options
+
+You will need to open a command prompt or terminal, change directory to where **qgroundcontrol.exe** is stored, and then run it. This is shown below for each platform (using the `--logging:full` option):
+
+Windows Command Prompt:
+```bash
+cd "\Program Files (x86)\qgroundcontrol"
+qgroundcontrol --logging:full
+```
+OSX Terminal app (**Applications/Utilities**):
+```bash
+cd /Applications/qgroundcontrol.app/Contents/MacOS/
+./qgroundcontrol --logging:full
+```
+Linux Terminal:
+```bash
+./qgroundcontrol-start.sh --logging:full
+```
+
+## Options
+The options/command line arguments are listed in the table below. 
+
+Option | Description
+--- | ---
+`--clear-settings` | Clears the app settings.
+`--logging:full` | Turns on full logging. See [Console Logging](https://docs.qgroundcontrol.com/en/SettingsView/console_logging.html#logging-from-the-command-line).
+`--logging:full,LinkManagerVerboseLog,ParameterLoaderLog` | Turns on full logging and turns off the following listed comma-separated logging options.
+`--logging:LinkManagerLog,PraameterLoaderLog` | Turns on the specified comma separated logging options
+`--unittest:name` | Runs the specified unit test. Leave off `:name` to run all tests.
```

Comment:
How does QGC know where the unit tests are/how are they run? I presume that this works only if you have just built QGC and you're starting QGC from its freshly built location? 
If so:
- Are the tests automatically built
- If the tests aren't present what happens with QGC. 
- Is QGC started along with the tests?

Reply:
Unit tests are built into debug builds automatically and are part of debug QGC. When you run a unit test it starts QGC but then QGC realizes it is running a unit test and does what the unit test wants instead of starting QGC normalls.

Reply:
OK, I'll put a note that this requires Debug version. What happens if you try it on a release version?

Reply:
Also are any of the other ones Debug-only too?